cheroot/connections.py

```diff
@@ -297,7 +296,7 @@ def _from_server_socket(self, server_socket):  # noqa: C901  # FIXME
             ssl_env = {}
             # if ssl cert and key are set, we try to be a secure HTTP server
             if self.server.ssl_adapter is not None:
-                try:
+                try:  # noqa: WPS505
```

Comment:
Why?

Reply:
Oh, this one is about nesting try blocks. Let's add a comment that we want it to be refactored into a separate function/method eventually. But again, I'd move the linting stuff in unrelated places to a separate PR so it doesn't distract us here.